src/weaveplots.jl

```diff
@@ -1,5 +1,8 @@
 # HACK: So Weave.jl has a submodule `WeavePlots` which is loaded using Requires.jl if Plots.jl is available.
 # This means that if we want to overload methods in that submodule we need to wait until `Plots.jl` has been loaded.
+import .Plots
+import .Weave
```

Comment:
This plotting kind of stuff with grabbing outputs and stuff is part of PlutoStaticHTML. I don't know whether PlutoStaticHTML can manage GIFs though. Pluto can, I think